### PR TITLE
[TDV-161] Improve limits management page design

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: src/components/User/Limits/components/Limits.Manager.vue [hunk 1]
```diff
@@ -73,21 +73,21 @@
...
+                    {{ type.replace('Out', '') }}
                   </span>
                   <input-field
-                    v-model="limits.payment[type]"
+                    :value="getLimitShownAmount(limits.payment[type])"
+                    @input="limits.payment[type] = $event || DEFAULT_MAX_AMOUNT"
                     class="limits-manager__limit-field"
                     :step="DEFAULT_INPUT_STEP"
-                    :label="' '"
+                    :placeholder="'Unlimited'"
```
Comment (max-nes) [inline src/components/User/Limits/components/Limits.Manager.vue]: why `:`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ for further information about branching and tagging conventions.
 ### Added
 -  Added is_coinpayments checkbox on asset creation page
 
+### Changed
+- Hide input placeholder only if label prop is empty
```
Comment (max-nes) [inline CHANGELOG.md]: Hide input placeholder only if label prop is _not_ empty

File: src/components/common/fields/InputField.vue [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
-    <input class="input-field__input"
+  <div
+    class="input-field"
+    :class="{'input-field--error': errorMessage}"
+  >
+    <input
+      class="input-field__input"
+      :class="{
+        'input-field__input--placeholder-overflowed': label,
+        ['text-align-' + align]: true
```
Comment (max-nes) [inline src/components/common/fields/InputField.vue]: what?

File: src/components/common/fields/scss/input.scss [hunk 4]
```diff
@@ -43,6 +43,10 @@
   &::placeholder {
     @include placeholder;
   }
+}
+
+.input-field__input--placeholder-overflowed {
```
Comment (max-nes) [inline src/components/common/fields/scss/input.scss]: `.input-field__input--placeholder-auto-hidden`

File: src/components/User/Limits/components/Limits.Manager.vue [hunk 5]
```diff
@@ -428,6 +353,9 @@
             this.filters.address = this.specificUserAddress
           }
         }
+      },
+      getLimitShownAmount (limit) {
```
Comment (max-nes) [inline src/components/User/Limits/components/Limits.Manager.vue]: 'normalizeLimitAmount'

File: src/components/User/Limits/components/Limits.Manager.vue (hunk 1 above)
Comment (ardier16) [reply src/components/User/Limits/components/Limits.Manager.vue]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (ardier16) [reply CHANGELOG.md]: Renamed

File: src/components/common/fields/InputField.vue (hunk 3 above)
Comment (ardier16) [reply src/components/common/fields/InputField.vue]: Removed align prop from input field

File: src/components/common/fields/scss/input.scss (hunk 4 above)
Comment (ardier16) [reply src/components/common/fields/scss/input.scss]: Renamed

File: src/components/User/Limits/components/Limits.Manager.vue (hunk 5 above)
Comment (ardier16) [reply src/components/User/Limits/components/Limits.Manager.vue]: Renamed

**Pull request merged**